src/librbd/image/RefreshParentRequest.h

```diff
@@ -4,7 +4,6 @@
 #ifndef CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 #define CEPH_LIBRBD_IMAGE_REFRESH_PARENT_REQUEST_H
 
-#include "include/int_types.h"
```

Comment:
You've yanked this from a lot of files -- where are the int types being defined?